src/components/Card.js

```diff
@@ -20,27 +34,83 @@ export default class Card {
     this._cardImage = this._cardElement.querySelector(".card__image");
     this._cardTitle = this._cardElement.querySelector(".card__title");
     this._likeButton = this._cardElement.querySelector(".card__like-button");
```

Comment:
Good job that you've found these variables only once here and defined them as **class fields**